.github/workflows/build_fork.yml

```diff
@@ -296,27 +293,18 @@ jobs:
 
       - name: test mathlib
         id: test
-        uses: liskin/gh-problem-matcher-wrap@v3
-        with:
-          linters: gcc
-          run: |
-            bash -o pipefail -c "
-            make -k -j 8 test"
+        run: |
+          bash -o pipefail -c "
+          make -k -j 8 test"
 
       - name: check for unused imports
         id: shake
-        uses: liskin/gh-problem-matcher-wrap@v3
```

Comment:
I think it might be working as intended for shake. Only the messages from lean itself are no longer handled.

Reply:
Restored.